try:
    raise ValueError
except ValueError:
    print("ValueError Exception!")
